sdk/scenario-format/src/interpret_trait.rs

```diff
@@ -19,6 +20,13 @@ impl InterpreterContext {
             ..self
         }
     }
+
+    pub fn with_missing_files(self, allow_missing_files: bool) -> Self {
```

Comment:
Let's call this allow_missing_files, just to be clear, and no argument.

Reply:
Done